auction9-frontend/src/app/services/auction-fetch.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuctionFetchService {
+
+  constructor(private http: HttpClient) { }
+
+  // endpoints
+  private readonly allActiveAuctionsEndpoint = 'http://127.0.0.1:3000/auctions/';
```

Comment:
Base url should be configurable. We will have different url when the apps are deployed to AWS.